table/block_based/block_based_table_reader.cc

```diff
@@ -3966,8 +3966,12 @@ uint64_t BlockBasedTable::ApproximateOffsetOf(const Slice& key,
                                               TableReaderCaller caller) {
   BlockCacheLookupContext context(caller);
   IndexBlockIter iiter_on_stack;
+
```

Comment:
This change will be brought into RocksDB Cloud as part of a merge of 6.7.3.

Reply:
Great!